hrms/patches/v14_0/update_repay_from_salary_and_payroll_payable_account_fields.py

```diff
@@ -0,0 +1,16 @@
+import frappe
+
+
+def execute():
+	if frappe.db.exists("Custom Field", {"name": "Loan Repayment-repay_from_salary"}):
+		frappe.db.set_value(
+			"Custom Field", {"name": "Loan Repayment-repay_from_salary"}, "fetch_if_empty", 1
+		)
+
+	if frappe.db.exists("Custom Field", {"name": "Loan Repayment-payroll_payable_account"}):
+		frappe.db.set_value(
+			"Custom Field",
+			{"name": "Loan Repayment-payroll_payable_account"},
+			"insert_after",
+			"payment_account",
+		)
```

Comment:
I think this was placed with the repay from salary checkbox since its visibility is dependent on toggling the checkbox. But moving it to the accounts section for context is also fine IG.